com.mojang.Minecraft.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.mojang.Minecraft.desktop</id>
+  <name>Minecraft</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>Minecraft is a game about placing blocks and going on adventures</summary>
+  <description>
+    <p>
+      A computer game, I should probably borrow some description from
+      wikipedia.
+    </p>
+  </description>
+  <url type="homepage">https://minecraft.net/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://lh3.googleusercontent.com/VONu2F1QvtTPBBgHvDZQbZBuierz1MIHGoQ9_QRsGpiu4ulrLQqeemJTboUI5Pnlcg=h900</image>
+    </screenshot>
+  </screenshots>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Mojang AB</developer_name>
```

Comment:
We're encouraging people to add [OARS](https://odrs.gnome.org/oars) metadata, it's really easy!

Reply:
Thanks, I added them, I also put a proper description.